frontend/sozisel-app/src/components/utils/Button/Button.tsx

```diff
@@ -0,0 +1,20 @@
+import "./Button.scss";
+
+import React, { ReactElement } from "react";
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  name: string;
+  type: "button" | "submit" | "reset" | undefined;
+}
+
+export default function Button({
+  name,
+  type,
+  ...props
+}: ButtonProps): ReactElement {
+  return (
+    <button {...props} type={type} className="button">
```

Comment:
And what if I want to use the default buttons styles mixed with my owns?

Reply:
You can pass `className` in props and overwrite the basic styles. It is possible that you have to use the `!important ` keyword in specific cases.